### ISSUE TITLE
conv3d in r3d_18: How to maintain the dimension?

### ISSUE THREAD

**Issue by u0251077** [pytorch/pytorch#50709]
### convolution in conv3d about padding

1. the input is (1, 3, 5, 112, 112)
2. the model is `models.video.r3d_18(pretrained=True, progress=False)`
3. the model summary 
```
VideoResNet(
  (stem): BasicStem(
    (0): Conv3d(3, 64, kernel_size=(3, 7, 7), stride=(1, 2, 2), padding=(1, 3, 3), bias=False)
    (1): BatchNorm3d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (2): ReLU(inplace=True)
  )
  (layer1): Sequential(
    (0): BasicBlock(
      (conv1): Sequential(
        (0): Conv3DSimple(64, 64, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1), bias=False)
        (1): BatchNorm3d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (2): ReLU(inplace=True)
      )
      (conv2): Sequential(
        (0): Conv3DSimple(64, 64, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1), bias=False)
        (1): BatchNorm3d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      )
      (relu): ReLU(inplace=True)
    )
    (1): BasicBlock(
      (conv1): Sequential(
        (0): Conv3DSimple(64, 64, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1), bias=False)
        (1): BatchNorm3d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (2): ReLU(inplace=True)
      )
      (conv2): Sequential(
        (0): Conv3DSimple(64, 64, kernel_size=(3, 3, 3), stride=(1, 1, 1), padding=(1, 1, 1), bias=False)
        (1): BatchNorm3d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
      )
      (relu): ReLU(inplace=True)
    )
  )
.............
```
4. input through the first layer in model
```
input = torch.zeros(1, 3, 5, 112, 112)
output = model.stem(input)
>>> torch.Size([1, 64, 5, 56, 56])
```
5. my question is : why the output is 1* 64* 5* 56* 56 
how to padding in pytorch
this is my Schematic diagram
![image](https://user-images.githubusercontent.com/17065425/104982826-6d20aa80-5a46-11eb-9d9d-c167bcae51fb.png)

**Comment by mruberry** [comment]
Try asking this question in our forums at https://discuss.pytorch.org/ first, @u0251077. We have an active community that can help you there.
